python/utils/README.md

```diff
@@ -0,0 +1,8 @@
+
+This directory contains utility scripts.
+
+- `generatejson.py`: Generates request json for the
```

Comment:
I'd indent these so they'll be easier to read later on.

      `generatejson.py`
       Generates request JSON for use with the [Cloud Vision API](https://cloud.google.com)
       given supplied parameters.  See the
       [Generating JSON Requests](https://cloud.google.com/vision/docs/concepts#generating_json_requests) within     
       the Cloud Vision API Concepts documentation for more information.